src/frequenz/channels/timer.py

```diff
@@ -187,11 +197,17 @@ class SkipMissedAndDrift(MissedTickPolicy):
         so is right on time (no drift) and the same happens for tick `T6.3`,
         which triggers at 6.3 seconds.
 
-        ```
+        <center>
+        ```bob
         0         1         2         3         4         5         6
-        o---------|-o-------|--o------|---------|--o------|--o------|--o--> time
-        T0          T1         T2.2                T3.2      T5.3      T6.3
+        *---------o-*-------|-o*------|-O-------|-o*------|--*------|--*--> time
```

Comment:
Oh, no, wait, I left those on purpose, just to mark the seconds units. In the other graphs, all the expected ticks are in exactly the second boundary, this is why there is no `|` (because I couldn't find any way to combine `|` and `o` in svgbob, otherwise that would have been the ideal).

You can even see the screenshot of how is it rendered in a comment above.

If you think it is not clear, then we could discuss how to change it, but it was left as it is intentionally.